codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -341,4 +370,12 @@ public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
         super.handlerRemoved(ctx);
         cleanup();
     }
+
+    public int getBlockSize() {
```

Comment:
lol, not putting `get` as a prefix to the (new) `currentBlockLength` method was my fault! Let me know which convention you would prefer